internal/checkers/suite_method_signature.go

```diff
@@ -0,0 +1,70 @@
+package checkers
+
+import (
+	"fmt"
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/ast/inspector"
+
+	"github.com/Antonboom/testifylint/internal/analysisutil"
+	"github.com/Antonboom/testifylint/internal/testify"
```

Comment:
Any reason to use tour local copy here, and directly use testify one un the generated tests?

See https://github.com/Antonboom/testifylint/pull/228#discussion_r1964239942

Reply:
1. This is not local copy. `internal/testify` contains domain (for linter context) entities, that absent in testify inself.
2. We cannot depend on `stretch/testify`, because it causes dependency bomb for linter's module. We should keep as min dependencies as possible.

Reply:
Make sense.

Maybe it could be explained in documentation (not sure where), if not already present.

I wondered why you had to bump testify to 1.10 in testdata recently, and previously wondered why you had testify in testdata

Reply:
added FAQ in contribution.md